exercises/concept/lucas-numbers/.formatter.exs

```diff
@@ -0,0 +1,5 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  line_length: 400
```

Comment:
How did that happen? 👀  Seems like a bit much

Reply:
I was playing around with getting the list on one line, I didn't remove it after. Oops